styles.css

```diff
@@ -1,20 +1,194 @@
 * {
```

Comment:
I'd put this and the normalize file into a css folder of their own to keep the root of your project nice and tidy